backend/src/xfd_django/xfd_api/api_methods/scan.py

```diff
@@ -171,15 +172,29 @@ def create_scan(scan_data: NewScan, current_user):  # pylint: disable=R0912, R09
                     detail="Scans with a date range must be single scans.",
                 )
 
-            # Validate ordering
+            # Validate ordering and format
             try:
                 parsed_start = parse_frontend_datetime(str(start_dt))
                 parsed_end = parse_frontend_datetime(str(end_dt))
+
                 if parsed_start >= parsed_end:
                     raise HTTPException(
                         status_code=400,
                         detail="start_datetime must be before end_datetime.",
                     )
+
+                # NEW: MAX_SCAN_DAYS-day validation
+                date_diff = parsed_end - parsed_start
```

Comment:
Logging: On both attempts and successes, log user id, request id, scan id, and any rejected fields as structured JSON. This will help now and in future Opentelemetry efforts so we have metrics to get